mu4/appshell/qml/HomePage/HomeMenu.qml

```diff
@@ -1,26 +1,74 @@
 import QtQuick 2.7
 import QtQuick.Controls 2.2
+import QtQuick.Layouts 1.3
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
 
 Rectangle {
-
     id: root
 
-    property var model: null
-
     signal selected(string name)
 
-    Column {
+    color: ui.theme.backgroundColor
+
+    RadioButtonGroup {
+        id: radioButtonList
+
         anchors.fill: parent
 
-        Repeater {
-            model: root.model
-            delegate: ItemDelegate {
-                height: 56
-                anchors.left: parent.left
-                anchors.right: parent.right
+        orientation: ListView.Vertical
+        spacing: 0
+
+        model: [
+            { "name": "scores", "title": "Scores", "icon": IconCode.MUSIC_NOTES },
+            { "name": "extensions", "title": "Add-ons", "icon":  IconCode.LIST_ADD },
+            { "name": "audio", "title": "Audio & VST", "icon":  IconCode.AUDIO },
+            { "name": "feautured", "title": "Featured", "icon":  IconCode.STAR },
+            { "name": "learn", "title": "Learn", "icon":  IconCode.GRADUATION_CAP },
+            { "name": "support", "title": "Support", "icon": IconCode.FEEDBACK },
+            { "name": "account", "title": "Account", "icon":  IconCode.ACCOUNT }
```

Comment:
Need to add string translatability (qsTrc)

Reply:
done